travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
+searched = {}
```

Comment:
@Adman should this `searched` thing be preserved between bot's restarts? If so, maybe using the storage might be a good idea.